### PR TITLE
DAT-3223 sort and add labels to template titles

### PR DESCRIPTION
**Pull request by dianafa**
https://wikia-inc.atlassian.net/browse/DAT-3223

### CODE REVIEW

File: extensions/VisualEditor/wikia/modules/ve/ui/dialogs/ve.ui.WikiaInfoboxInsertDialog.js [hunk 1]
```diff
@@ -178,6 +179,27 @@ ve.ui.WikiaInfoboxInsertDialog.prototype.showResults = function ( data ) {
 };
 
 /**
+ * Sort template titles alphabetically
+ *
+ * @param array of infoboxes
+ */
+ve.ui.WikiaInfoboxInsertDialog.prototype.sortTemplateTitles = function ( infoboxes ) {
```
Comment (RafLeszczynski) [inline extensions/VisualEditor/wikia/modules/ve/ui/dialogs/ve.ui.WikiaInfoboxInsertDialog.js:256]: please consider creating unit test for this method

**Comment by RafLeszczynski** [comment]
LGTM but please think about unit tests :-)

File: extensions/wikia/PortableInfobox/controllers/ApiQueryAllinfoboxes.class.php [hunk 2]
```diff
@@ -34,4 +35,19 @@ public function execute() {
 	public function getVersion() {
 		return __CLASS__ . '$Id$';
 	}
+
+	/**
+	 * @desc As a infobox template label we want to return a nice, clean text, without e.g. '_' signs
+	 * @param $text infobox template title
+	 * @return String
+	 */
+	private function createLabel( $text ) {
```
Comment (sebastianmarzjan) [inline extensions/wikia/PortableInfobox/controllers/ApiQueryAllinfoboxes.class.php:44]: Assuming we don't have code that already does this in the repo, LGTM. Did you check?

Comment (dianafa) [inline extensions/wikia/PortableInfobox/controllers/ApiQueryAllinfoboxes.class.php:44]: yes, I was checking before :)

**Pull request merged**